README.rst

```diff
@@ -109,3 +109,12 @@ By default the ELBs will have a security group opening them to the world on 80 a
 If you set the protocol on an ELB to HTTPS you must include a key called `certificate_name` in the ELB block (as example above) and matching cert data in a key with the same name as the cert under `ssl` (see example above). The `cert` and `key` are required and the `chain` is optional.
 
 The certificate will be uploaded before the stack is created and removed after it is deleted.
+
+Includes
+++++++++++
+If you wish to include some static cloudformation json and have it merged with the template generated by bootstrap-cfn. You can do the following in your template yaml file::
+
+    includes:
+      - /path/to/cloudformation.json
+
+The tool will then merge this with the generated template *overwriting* any keys in the original template that clash.
```

Comment:
Will try to make it clearer. Quite hard to explain deep merging in English though!